dsc_lib/src/functions/path.rs

```diff
@@ -31,8 +31,15 @@ impl Function for Path {
         debug!("Executing path function with args: {:?}", args);
 
         let mut path = PathBuf::new();
+        let mut first = true;
         for arg in args {
             if let Value::String(s) = arg {
+                // if first argument is a drive letter, add it with a separator suffix as PathBuf.push() doesn't add it
+                if first && s.len() == 2 && s.chars().nth(1).unwrap() == ':' {
```

Comment:
On Unix `~/a:/b:/c:/d` is a valid path too. Will this change work with multiple `:` ?

Add a test for multiple `:` in path.

Reply:
I'll add that test, but the behavior will be different between Windows and non-Windows